### PR TITLE
Merge Dev into Master -  Be able to add any config to squid.conf 

### PR DESCRIPTION
**Pull request by iblackman**
## Description
This feature includes the possibility to add any config in squid.conf not covered by the variables already created in the role.

## Motivation and Context
The role allows to add some configs such as: 
* `acl`; 
* `http_access`;
* `http_port`;
* `coredump_dir`;
* etc.
But there are many other that the role doesn't cover. For example:
* `cache_mem`;
* or a second `cache_dir`.

## How Has This Been Tested?
It was tested with: 
* `Molecule` 2.12.1
*  `Docker` 18.03.0-ce
*  `Vagrant` 2.0.2
* `VirtualBox` 5.1.34

There are 2 scenarios in `Molecule` using Centos7.4:
* with Docker
* with Vagrant

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: molecule/resources/playbooks/playbook.yml [hunk 1]
```diff
@@ -9,3 +9,7 @@
     - name: execute role
       include_role:
         name: stone-payments.squid
+      vars:
+        squid_custom_configs:
+          - "cache_mem 128 MB"
+        squid_diskcache: "aufs {{ squid_coredumpdir }} 1000 16 256"
```
Comment (bcdonadio) [inline molecule/resources/playbooks/playbook.yml]: This makes absolutely no sense. Don't ever mix a folder being served to third-parties with coredumps.

File: templates/squid.conf.j2 [hunk 2]
```diff
@@ -56,7 +56,13 @@ cache_dir {{ squid_diskcache }}
 
 {% if squid_coredumpdir %}
 coredump_dir {{ squid_coredumpdir }}
```
Comment (bcdonadio) [inline templates/squid.conf.j2]: This is not a useful directive. It's rarely used and probably shouldn't be here.

Please, research on what is a coredump.

**Review by bcdonadio — CHANGES_REQUESTED** [review]
Please, fix the confusion between the squid_cache folder and the coredump target.

**Comment by iblackman** [comment]
Fixed.

**Pull request merged**